charts/connect/values.yaml

```diff
@@ -70,6 +70,17 @@ connect:
   # Additional annotations to be added to the service resource
   serviceAnnotations: {}
 
+  # 1Password Operator Service Account Configuration
+  serviceAccount:
+    # The name of the 1Password Conenct Operator
+    create: "{{ .Values.connect.create }}"
```

Comment:
Let's make it `false` by default, so existing deployments are not affected when upgrading to a new version. With that we keep it backward compatible.